### ISSUE TITLE
[💡 Feature Request]: Be able to play music using a computer browser?

### ISSUE THREAD

**Issue by Darkov3** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#892]
### What's the Problem?

This is the main reason I wanted to try this out

### What's your solution or idea?

Should be technically possible, yes?

### Alternatives Considered

-

### App Version

v1.0.0-rc.2

### Pre-submission Checklist

- [X] I have checked [the issue tracker](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues?q=is%3Aissue+label%3A%22%E2%9C%A8+enhancement%22+) and didn't find any duplicate issue
- [X] This issue only covers one particular feature request

**Comment by sumeshir26** [comment]
No, this is not possible as of now, due to the technical limitations of the Google Assistant Web SDK
